posthog/api/property_definition.py

```diff
@@ -48,8 +68,13 @@ def get_queryset(self):
                     names = tuple(properties_to_filter.split(","))
                     name_filter = f"AND name IN %(names)s"
                 else:
-                    name_filter = ""
+                    search = self.request.GET.get("search", None)
                     names = ()
+                    name_filter = (
+                        ""
+                        if search is None
```

Comment:
Maybe we want to check for falsy values in general? Not sure what happens if you search by an empty string.